recipes/phyloacc/run_test.sh

```diff
@@ -28,6 +28,19 @@ for file in "${files[@]}"; do
 done
 echo " ** TEST DATA DOWNLOAD OK."
 
+# echo " ** BEGIN BINARY TEST."
+# if ! PhyloAcc-ST; then
```

Comment:
What's the purpose of this newly added code if it is commented out ?

Reply:
This was to help me debug the problem with the previous build. However there's no good way to test the binaries directly so I had to comment it out for the actual build. I'd like to keep it in case I need to debug a similar situation in the future.